internal/mining/mining.go

```diff
@@ -148,23 +228,23 @@ const (
 	// incoming non vote transactions before template regeneration
 	// is required.
 	templateRegenSecs = 30
+
+	// maxResorts is the maximum number of times a transaction can be
+	// added back to the priority queue due to the fee changing.
+	maxResorts = 10
```

Comment:
How did you decide on `10` for this?

Reply:
Removed. The scenario of having transaction chains reprioritize depending on their ancestors should just be 0 for the sake of simplicity.